documentation/docs/60-appendix/05-integrations.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Integrations
+---
+
+## Preprocessors
+
+Preprocessors transform your `.svelte` files before passing them to the compiler. E.g. if your `.svelte` file uses TypeScript and PostCSS, it must first be transformed into JavaScript and CSS so that the Svelte compiler can handle it. There are many [available preprocessors](https://sveltesociety.dev/tools#preprocessors).
```

Comment:
```suggestion
Preprocessors transform your `.svelte` files before passing them to the compiler. For example, if your `.svelte` file uses TypeScript and PostCSS, it must first be transformed into JavaScript and CSS so that the Svelte compiler can handle it. There are many [available preprocessors](https://sveltesociety.dev/tools#preprocessors).
```